File: app/src/main/java/cs407/socialkarmaapp/Comment.java
package cs407.socialkarmaapp;

public class Comment {
    private String author, authorName, comment, postId;
    private int timestamp, vote;

    public Comment() {

    }

    public String getAuthor() {
        return author;
    }

    public String getAuthorName() {
        return authorName;
    }

    public String getComment() {
        return comment;
    }

    public String getPostId() {
        return postId;
    }

    public int getTimestamp() {
        return timestamp;
    }

    public int getVote() {
        return vote;
    }
}

File: app/src/main/java/cs407/socialkarmaapp/Models/Comment.kt
package cs407.socialkarmaapp.Models

class Comment() {
    var postCommentId: String = ""
    var author: String = ""
    var authorName: String = ""
    var postId: String = ""
    var comment: String = ""
    var timestamp: Int = 0
    var votes: Int = 0
    var voted: Int = 0

    constructor(postCommentId: String, author: String, authorName: String, postId: String, comment: String, timestamp: Int, votes: Int, voted: Int) : this() {
        this.postCommentId = postCommentId
        this.author = author
        this.authorName = authorName
        this.postId = postId
        this.comment = comment
        this.timestamp = timestamp
        this.votes = votes
        this.voted = voted
    }
}